verbsense/src/main/java/edu/illinois/cs/cogcomp/verbsense/data/PropbankReader.java

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
There is a class of the same name in corpusreaders -- if this is duplicate, use the original; otherwise, rename